backon/src/backoff/mod.rs

```diff
@@ -12,3 +12,24 @@ pub use fibonacci::FibonacciBuilder;
 mod exponential;
 pub use exponential::ExponentialBackoff;
 pub use exponential::ExponentialBuilder;
+
+trait Random {
```

Comment:
Hi, I don't want to introduce a new trait for this. Maybe we can store a new `fastrand::Rng` in every `FibonacciBackoff`?